registration/validators.py

```diff
@@ -69,3 +70,12 @@ def validate_client_interaction(interaction):
 def validate_interactions(interactions):
     for interaction in interactions.values():
         validate_client_interaction(interaction)
+
+
+def validate_field_order(field_order, role):
+    Field = apps.get_model("registration", "Field")
+    if (
+        not field_order
```

Comment:
is there a reason why `not` is not outside the brackets :o

Reply:
oh, didn't even notice the linter did that